app/views/layouts/application.html.erb

```diff
@@ -36,7 +36,6 @@
           } catch {}
         })
       </script>
-      <script async src="https://cdn.headwayapp.co/widget.js"></script>
```

Comment:
Is this going to break this sparkles link?

![CleanShot 2025-06-26 at 12 52 36@2x](https://github.com/user-attachments/assets/a22fbb03-053b-43f9-b690-69a37fb97bea)

Reply:
We can probably drop this code as well https://github.com/hackclub/hcb/blob/d293e57763729f69a1612606e22095761650eccd/app/views/layouts/application.html.erb#L28-L38